actors/evm/src/interpreter/transaction.rs

```diff
@@ -1,788 +0,0 @@
-#![allow(dead_code)]
-
-use {
-    crate::interpreter::{H160, H256, U256},
-    bytes::Bytes,
-    fil_actors_runtime::ActorError,
-    fvm_shared::crypto::signature::SECP_PUB_LEN,
-    rlp::{DecoderError, Rlp, RlpStream},
-    sha3::{Digest, Keccak256},
-    std::{fmt::Debug, ops::Deref},
-};
-
-#[derive(Debug, PartialEq, Eq, Clone)]
-pub enum TransactionAction {
-    Call(H160),
-    Create,
-}
-
-#[derive(Debug, PartialEq, Eq)]
-pub struct AccessListItem {
-    pub address: H160,
-    pub slots: Vec<H256>,
-}
-
-pub enum Transaction {
-    Legacy {
-        chain_id: Option<u64>,
-        nonce: u64,
-        gas_price: U256,
-        gas_limit: u64,
-        action: TransactionAction,
-        value: U256,
-        input: Bytes,
-    },
-    EIP2930 {
-        chain_id: u64,
-        nonce: u64,
-        gas_price: U256,
-        gas_limit: u64,
-        action: TransactionAction,
-        value: U256,
-        input: Bytes,
-        access_list: Vec<AccessListItem>,
-    },
-    EIP1559 {
-        chain_id: u64,
-        nonce: u64,
-        max_priority_fee_per_gas: U256,
-        max_fee_per_gas: U256,
-        gas_limit: u64,
-        action: TransactionAction,
-        value: U256,
-        input: Bytes,
-        access_list: Vec<AccessListItem>,
-    },
-}
```

Comment:
Removing this here should be fine because the onus of native Ethereum transaction validation falls on the account actor via account abstraction. It would've never been used here. Our EVM runtime also doesn't care about Access Lists and Ethereum gas implications. And Filecoin transations can be assimilated to EIP-1559 transactions in terms of gas data model.

Reply:
Well, the only issue is that we'll need to resurect this code when we _do_ implement the abstract account.... cc @mriise

Reply:
we'll probably need a very small part of it, just signature verification.